epub33/rs/index.html

```diff
@@ -1000,28 +1000,28 @@ <h4>The <code>rendition:layout</code> Property</h4>
 				<section id="orientation">
 					<h4>The <code>rendition:orientation</code> Property</h4>
 
-					<p>The default global value is <code>auto</code> if no <code>meta</code> element carrying this
+					<p id="fxl-orientation_default" data-tests="fxl-orientation_default">The default global value is <code>auto</code> if no <code>meta</code> element carrying this
```

Comment:
Both the ID and test should have hyphen instead of underscore here. (Never underscores in the ID, please.)

Applies throughout.